SA3/qa_web/templates/qa_web/base.html

```diff
@@ -7,36 +7,67 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta name="description" content="">
-    <meta name="author" content="">
+    <meta name="author" content="SA3 Group">
+    {% block meta %}
+    {% endblock %}
 
     <title>Got Answers </title>
 
     <!-- Latest compiled and minified CSS -->
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <!-- header -->
+    <link rel="stylesheet" href="{% static 'css/pagedown.css' %}" />
+    <link rel="stylesheet" type="text/css" href="{% static 'css/bootstrap.min.css' %}" />
+    <link rel="stylesheet" type="text/css" href="{% static 'css/one-page-wonder.min.css' %}" />
 
     <!-- Custom fonts for this template -->
     <link href="https://fonts.googleapis.com/css?family=Catamaran:100,200,300,400,500,600,700,800,900" rel="stylesheet">
     <link href="https://fonts.googleapis.com/css?family=Lato:100,100i,300,300i,400,400i,700,700i,900,900i" rel="stylesheet">
+    <link href="https://use.fontawesome.com/releases/v5.0.6/css/all.css" rel="stylesheet">
 
     <!-- Custom styles for this template -->
     <link href="{% static 'css/one-page-wonder.min.css' %}" rel="stylesheet">
+    {% block styles %}
+    {% endblock %}
     
   </head>
 
   <body>
-    <div>
-        {% block content %}
-        {% endblock %}
-    </div>
+    {% include "qa_web/includes/navbar.html" %}
 
-    <!-- JQuery from BootstrapCDN -->
-    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+<header class="masthead text-center text-white">
+    {% block header %}
 
+
+    {% endblock %}
+      <div class="bg-circle-1 bg-circle"></div>
+      <div class="bg-circle-2 bg-circle"></div>
+      <div class="bg-circle-3 bg-circle"></div>
+      <div class="bg-circle-4 bg-circle"></div>
+</header>
+
+<div align="center" style="vertical-align:bottom" class="masthead-content">
+    {% block content %}
+
+    {% endblock %}
+</div>
+
+
+    {% include "qa_web/includes/footer.html" %}
+
+
+
+
+    <!-- JQuery -->
+    <script src="https://code.jquery.com/jquery-3.3.1.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
```

Comment:
I feel like the js can be loaded in the `<head>` tag so we could see all of our static files in them same place for convenience. But if this is a good practice and our features work then we can definitely keep it this way.